src/Models/Context/DeeplinkContext/DeeplinkContext.tsx

```diff
@@ -79,12 +79,14 @@ export const DeeplinkContextReducer: (
                                 AzureResourceDisplayFields.url
                             )
                     );
-                    draft.adtResourceId = itemInLocalStorage.id;
-                    const resourceFromItem = getResourceFromEnvironmentItem(
-                        itemInLocalStorage,
-                        AzureResourceTypes.DigitalTwinInstance
-                    ) as IADTInstance;
-                    setSelectedAdtInstanceInLocalStorage(resourceFromItem);
+                    if (itemInLocalStorage) {
```

Comment:
This makes us null safe, but will the app respond effectively if we don't find the value in storage?

Reply:
yep, it will just try to fetch the id (the long flow)

Reply:
great. Glad we caught that.